File: class01/java/AP0103-MediaDuasNotas.java
import java.util.Scanner;
public class Media {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Nota 1: ");
        double n1 = sc.nextDouble();
        System.out.print("Nota 2: ");
        double n2 = sc.nextDouble();
        double media = (n1 + n2) / 2;
        System.out.printf("Media: %.1f%n", media);
        if (media >= 7) System.out.println("Aprovado");
        else System.out.println("Reprovado");
    }
}

File: class01/cpp/AP0103-MediaDuasNotas.cpp
#include <iostream>
#include <iomanip>
int main() {
    double n1, n2;
    std::cout << "Nota 1: "; std::cin >> n1;
    std::cout << "Nota 2: "; std::cin >> n2;
    double media = (n1 + n2) / 2;
    std::cout << "Media: " << std::fixed << std::setprecision(1) << media << std::endl;
    if (media >= 7) std::cout << "Aprovado" << std::endl;
    else std::cout << "Reprovado" << std::endl;
    return 0;
}
